lib/logster/redis_store.rb

```diff
@@ -272,6 +272,12 @@ def clear_all
       @redis.del(env_key)
       @redis.del(grouping_key)
       @redis.del(solved_key)
```

Comment:
tricky tricky .... "clear all" button in the UX should not nuke patterns, so you need this thing to take a param in to say "clear patterns as well"

Reply:
Maybe I'm misunderstanding here, but this method is not called when you hit "clear all" in the UX. Instead [`Store#clear`](https://github.com/discourse/logster/blob/5bc5014f66b059becf92508f56214abb7ef2f859/lib/logster/redis_store.rb#L233) is called:

https://github.com/discourse/logster/blob/5bc5014f66b059becf92508f56214abb7ef2f859/lib/logster/middleware/viewer.rb#L86-L91

I did a quick search and this method is only used in tests.

But I think it's worth adding a test case for this particular case to make sure we don't regress.